### PR TITLE
Adding cluster-role and role permission required for the chaos-operator:2.11.0.

### PR DESCRIPTION
**Pull request by Adarshkumar14**
#### What this PR does / why we need it:
- To run chaos experiments in Openshift cluster, these permissions are required by the  chaos-operator:2.11.0 to run the experiment pods.
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] DCO signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

**Comment by Jasstkn** [comment]
@Adarshkumar14 hi. could you double-check docs update? looks like something wrong with it and you have more spaces than it should be

File: charts/litmus/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # litmus
 
-![Version: 2.11.1](https://img.shields.io/badge/Version-2.11.1-informational?style=flat-square) ![AppVersion: 2.11.0](https://img.shields.io/badge/AppVersion-2.11.0-informational?style=flat-square)
+![Version: 2.11.2](https://img.shields.io/badge/Version-2.11.2-informational?style=flat-square) ![AppVersion: 2.11.0](https://img.shields.io/badge/AppVersion-2.11.0-informational?style=flat-square)
```
Comment (Jasstkn) [inline charts/litmus/README.md:3]: at the end of the line

**Review by Jasstkn — APPROVED** [review]

**Review by imrajdas — APPROVED** [review]

**Comment by Jasstkn** [comment]
@Adarshkumar14 hi. there are merge conflicts. Could you resolve it and I'll handle the rest?

**Comment by Jasstkn** [comment]
@Adarshkumar14 hi! looks like you accidentally erased your changes.

**Comment by Adarshkumar14** [comment]
> @Adarshkumar14 hi! looks like you accidentally erased your changes.
Hi @Jasstkn,
@Jonsy13  had added these changes in his release update PR. We can close this PR.

**Pull request closed without merging**